### DISCUSSION TITLE
images wont safe

### DISCUSSION THREAD

**Discussion by melikootje** [givanz/Vvveb#40] [Q&A]
goodmorning,

now the installation went well, without any errors.. but once i upload a picture, i get these errors

(https://i.postimg.cc/mZ35w4Dt/Screenshot-20230905-085705.png)

what does it mean & how do i fix it?

**Comment by givanz** [comment]
Hi

This error is caused by nginx having a low value set for `client_max_body_size` default is 1MB you need to edit your nginx config and increase the value or if it's not set add it with a high value like 20M.

https://stackoverflow.com/questions/28476643/default-nginx-client-max-body-size

**Comment by melikootje** [comment]
hi, now this is fixed i cannot add a font of any of the listed google fonts.. i tried adding it in the home template html file by adding google fonts stylesheet but that wont work either

**Comment by givanz** [comment]
Thanks for the bug report, to fix it replace `vvveb/public/js/vvvebjs/builder.js` with this file https://demo.vvveb.com/js/vvvebjs/builder.js you might need to clear browser cache.

The bug that I was able to reproduce was that the font is selected but on page save it's lost.

![Screenshot_2023-09-05_11-58-58](https://github.com/givanz/Vvveb/assets/302738/98c2e918-630a-40a9-9fc2-138dbaf33f9a)
